fetch.bs

```diff
@@ -1091,6 +1127,20 @@ Unless stated otherwise, it is unset.
 "<code>follow</code>", "<code>error</code>", or "<code>manual</code>".
 Unless stated otherwise, it is "<code>follow</code>".
 
+<div class="note no-backref">
+ <dl>
+  <dt>"<code>follow</code>"
+  <dd>Used to follow all redirects incurred when fetching a resource.
+
+  <dt>"<code>error</code>"
+  <dd>This mode ensures that fetch throws an error when a request for a resource is redirected.
```

Comment:
It might make more sense to talk about the response being a redirect. Since no redirect takes place. Same below.